.dockerignore

```diff
@@ -15,6 +15,8 @@ msg/*Result.msg
 msg/_*.py
 build_isolated/
 devel_isolated/
+.git
+./src/magellan_vision
```

Comment:
Eventually you'll probably want to ignore a more specific path with just the large resources in this directory (assuming something in this package gets run on the robot). Not a big deal now

Reply:
I'll keep this in mind down the road